pytrip/__init__.py

```diff
@@ -24,6 +24,7 @@
 # flake8: noqa
 
 import logging
+from pytrip.cube import Cube
```

Comment:
issue #308 could be left open for handling the trip .SVV cubes, but this is not important now.